### ISSUE TITLE
Scan local files

### ISSUE THREAD

**Issue by bmarsh9** [SAP/credential-digger#88]
Question. How could I use credential-digger to scan a local file that has already been downloaded. Basically looking to use it like this:

```
from credential-digger import scan

results = scan("/path/to/file or directory",*args)
```

I was looking through the code and I wasn't sure where a good entry point was. Thanks!

**Comment by marcorosa** [comment]
Hi @bmarsh9, thank you for your suggestion. Indeed, we are already working on that feature, and it will be part of the next release ⏳

**Comment by bmarsh9** [comment]
Cool, thanks!

**Comment by marcorosa** [comment]
Feature merged on the main branch, and released with v3.1. I close the issue.
